### DISCUSSION TITLE
Using rabbitmq in openmage

### DISCUSSION THREAD

**Discussion by albertobraschi** [OpenMage/magento-lts#2551] [Questions & Answers] [answered]
Hello everybody!

Is there any way to use rabbitmq in openmage?

**Comment by Flyingmana** [comment]
how would you want to use it, or what would you expect it provide?

**Comment by albertobraschi** [comment]
i would like to use it, rabbitmq, to manage cronjobs
like in magento2

**Reply by Flyingmana** [reply]
so you search for something, which puts the now needed cronjobs into rabbitmq, and have rabbitmq as the trigger to run the cronjobs on possibly various server nodes decentralized?
Do you have a link where for Magento2 this is documented? (I was not aware they support something like this now)

I think something like this was not yet made for OpenMage

**Reply by sreichel** [reply, marked as answer]
M2: [RabbitMQ | Adobe Commerce Developer Guide](https://devdocs.magento.com/guides/v2.4/install-gde/prereq/install-rabbitmq.html)

Maybe this is something to start with ... https://github.com/smart-devs/FireGento_RabbitMQ

(DDEV: https://github.com/drud/ddev-contrib/tree/master/docker-compose-services/rabbitmq)
